Source/WebCore/editing/TextIteratorBehavior.h

```diff
@@ -69,6 +69,9 @@ enum class TextIteratorBehavior : uint16_t {
 #if ENABLE(ATTACHMENT_ELEMENT)
     EmitsObjectReplacementCharactersForAttachments = 1 << 13,
 #endif
+
+    // Used by AX to expose untransformed kana text.
```

Comment:
It might be good to spell out "accessibility" instead of "AX" (I've found some folks don't know that short-hand, and this change is not in the `accessibility` subfolder so might be good not to assume knowledge)

Reply:
Good idea, I can update this!

Reply:
Yes, AX is a somewhat Apple-internal jargon, related to the AX API... Short for Accessibility in the same way that NS is short for Next Step, but how many people know that? 😅